packages/remark-plugins/plugins/paragraph-custom-alerts/index.js

```diff
@@ -9,7 +9,43 @@ const sigils = {
   '!>': 'danger',
 }
 
-export default function paragraphCustomAlertsPlugin() {
+function generateAlertElement({ format, type, children }) {
+  switch (format) {
+    case 'html': {
+      return {
+        type: 'wrapper',
+        children: [children],
+        data: {
+          hName: 'div',
+          hProperties: {
+            className: ['alert', `alert-${type}`, 'g-type-body'],
+          },
+        },
+      }
+    }
+    default:
+    case 'jsx': {
+      return u(
+        'mdxJsxFlowElement',
+        {
+          name: 'div',
+          attributes: [
+            u(
+              'mdxJsxAttribute',
+              { name: 'className' },
+              `alert alert-${type} g-type-body`
+            ),
+          ],
+        },
+        [children]
+      )
+    }
+  }
+}
```

Comment:
Added this to support our existing `markdown-to-html` utility, which rightfully doesn't want jsx nodes.